docs/datagrid/datagrid-v2-migration.md

```diff
@@ -0,0 +1,33 @@
+Upgrade DataGrid to v2
+===
+The core of DataGrid has been changed in the version 2.0. We increased the peformance, flexibility and added accessibility supoort. It means that API options have been changed and definition of styles or data source are different. We described all changes below. 
```

Comment:
2.0? It will be Dashboards v3.0 I guess, I don't know if we have somewhere an external DataGrid versioning.

Reply:
Hm...but we need to distinguish which version is critical. 

Any ideas? @karolkolodziej @ddragula

Reply:
@stitot any suggestions?